app/src/main/java/org/wikipedia/LauncherController.kt

```diff
@@ -0,0 +1,57 @@
+package org.wikipedia
+
+import android.content.ComponentName
+import android.content.Context
+import android.content.pm.PackageManager
+import org.wikipedia.settings.Prefs
+
+object LauncherController {
+
+    fun setIcon(icon: LauncherIcon) {
+        val context = WikipediaApp.instance.applicationContext
+        val packageManager = context.packageManager
+        LauncherIcon.entries.forEach { i ->
```

Comment:
Can we rename the `i` to `icon` or `launcherIcon`?

Reply:
done.